easybuild/easyconfigs/b/binutils/binutils-2.25-GCC-4.9.3-binutils-2.25.eb

```diff
@@ -6,7 +6,7 @@ version = '2.25'
 homepage = 'http://directory.fsf.org/project/binutils/'
 description = "binutils: GNU binary utilities"
 
-toolchain = {'name': 'GCC', 'version': '4.9.3-binutils-%(version)s'}
+toolchain = {'name': 'GCC', 'version': '4.9.3-binutils-%s' % version}
```

Comment:
Unless we can't fix https://github.com/hpcugent/easybuild-framework/issues/1402, this should be restored back to using `%(version)s`, indeed.